javascript/rules/stylistic.md

```diff
@@ -0,0 +1,18 @@
+
+## Configuration Selected
+We have selected the [default](https://eslint.style/packages/js) rule set.  Deviations from the rule set will be discussed below.
```

Comment:
Should this have a link to stylistic instead?

Reply:
this is a link to stylistic?

Reply:
Oh oops I got confused cause its still `eslint`, but yeah ignore this comment!